src/install-kani.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# If version is latest, install directly from cargo
+if [ "$1" == "latest" ]; then
+    cargo install --locked kani-verifier;
+else
+    VERSION=$1
+    cargo install --version $VERSION --locked kani-verifier;
+fi
+
+# Check exit status for error handling
+if [ $? -ne O ]; then
+    echo "Error installing Kani. Please check if the provided version is correct"
```

Comment:
Maybe use the github format message:

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message